### PR TITLE
Cards added

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Please review this cards layout made by me thanks.
live-https://loving-shockley-e39c7a.netlify.app

### CODE REVIEW

File: components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+.card__image-container {
+    width: 11rem;
+    height: 11rem;
+
+}
+
+.card__image-container>img {
+    border-radius: px4px 4px 0 0;
```
Comment (mikesch803) [inline components/cards/cards.css]: typo

Comment (partha8) [inline components/cards/cards.css]: I think there's a typo here.

Comment (mikesch803) [reply components/cards/cards.css]: also line 31 is empty

Comment (Dev-AkshatGupta) [reply components/cards/cards.css]: Oh! yeah have not seen it thanks man
Thanks for reviewing it too

Comment (Dev-AkshatGupta) [reply components/cards/cards.css]: Oh! yeah have not seen it thanks man
Thanks for reviewing it too

File: index.html [hunk 2]
```diff
@@ -12,8 +12,149 @@
...
+                <div class="card-element__bottom">
+                    <button class="card-button "><i class="far fa-thumbs-up"></i></button>
+                    <button class="card-button "><i class="fas fa-share"></i></button>
+                    <button class="card-button "><i class="far fa-bell"></i></button>
+                </div>
+            </div>
+        </div>
+
+        <h2>card with dismiss and shadow</h2>
+        <div style="margin: 50px;">
```
Comment (vrx29) [inline index.html:107]: try to avoid inline styling

File: components/cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+.card-vertical-upper-text {
+    margin-left: 1rem;
+    margin-top: .5rem;
+}
+
+.card-vertical-img-cont {
+    width: 35rem;
+    height: 19.6;
```
Comment (partha8) [inline components/cards/cards.css]: 19.6rem or is it just unit less?

**Comment by partha8** [comment]
Everything else looks great and has cool pictures!

File: index.html (hunk 2 above)
Comment (Dev-AkshatGupta) [reply index.html:107]: Thanks for reviewing it buddy its just for now reviewing purpose will trash all html its just for clarity in peers review

File: components/cards/cards.css (hunk 3 above)
Comment (Dev-AkshatGupta) [reply components/cards/cards.css]: Checked it man it was a big mistake by me resolved it Thanks again

**Comment by Dev-AkshatGupta** [comment]
> Everything else looks great and has cool pictures!

Thank you buddy :)

**Comment by Dev-AkshatGupta** [comment]
Been reviewed and also resolve all the comments.
Thanks to reviewers

**Pull request merged**